app/code/Magento/InventoryCatalogSearch/Test/Integration/Plugin/Model/Search/FilterMapper/TermDropdownStrategy/ApplyStockConditionToSelectTest.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryCatalogSearch\Test\Integration\Plugin\Model\Search\FilterMapper\TermDropdownStrategy;
```

Comment:
We don't create tests on plugin
We create tests only on functionality

Reply:
Fixed.